src/cli/javascript/request_filter.rs

```diff
@@ -52,13 +63,13 @@ impl RequestFilter {
                         request
                             .url_mut()
                             .set_path(js_response.headers["location"].as_str());
-                        self.on_request(request).await
+                        self.on_request(request, None).await
```

Comment:
If we pass `None` here and have a default filter above, then we will be stuck in an infinite loop.

Reply:
yes, infinite loop.
Should we have empty string for http filter?